scripts/aexpect_helper

```diff
@@ -119,11 +118,11 @@ def main():     # too-many-* pylint:disable=R0914,R0912,R0915
 
             # Write shell PID to file
             server_log.info('Writing shell PID file %s', shell_pid_filename)
-            with open(shell_pid_filename, "w") as file_obj:
+            with open(shell_pid_filename, "w", encoding="utf-8") as file_obj:
```

Comment:
Ah, an option for open that I didn't know about. Living and learning.

Reply:
Well in py2 one had to use codecs to open with encoding, in py3 they added it as an argument and in 3.10 they added `encoding="locale"` which should be what we use in avocado (`locale.getpreferredencoding()`)